packages/picasso/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -232,11 +235,23 @@ export const RichTextEditor = forwardRef<HTMLDivElement, Props>(
         {counterMessage && (
           <Counter error={counterError} message={counterMessage} />
         )}
+        {hiddenInputId && enableFocusOnLabelClick(hiddenInputId)}
       </Container>
     )
   }
 )
 
+const hiddenInputStyle: React.CSSProperties = {
+  position: 'absolute',
+  opacity: 0,
```

Comment:
nit:
```suggestion
  visibility: hidden,
```
would it work?

Reply:
@denieler nope, neither `visiblity: hidden` or `display: none` works. Browser is too clever to focus not visible element ;-)